src/microbe_stage/Patch.cs

```diff
@@ -229,7 +228,6 @@ public void WriteToArchive(ISArchiveWriter writer)
         writer.WriteObject(BiomeTemplate);
         writer.WriteObject(currentSnapshot);
         writer.WriteObject(gameplayPopulations);
-        writer.WriteObject(activeWorldEffectVisuals);
```

Comment:
You've removed a write here, but you haven't modified the version number nor adjusted the read so that it does a dummy read of this data and then discards the thing it read.